specs/notation-cli.md

```diff
@@ -28,15 +29,16 @@ Usage:
 
 Available Commands:
   certificate Manage certificates in trust store
-  inspect     Inspect all signatures associated with the signed artifact
   key         Manage keys used for signing
-  list        List signatures of the signed artifact
-  login       Login to registry
-  logout      Log out from the logged in registries
+  list        List signatures of a signed OCI artifact
+  login       Log into OCI registries
+  logout      Log out from the logged in OCI registries
   plugin      Manage plugins
   policy      Manage trust policy configuration for signature verification
-  sign        Sign artifacts
-  verify      Verify artifacts
+  sign        Sign OCI artifacts
+  verify      Verify OCI artifacts
+  blobs       Sign, verify and inspect singatures associated with blobs
```

Comment:
Elsewhere in the changes this command is stated to be `blob` not `blobs`. Which is correct? Also "signature" appears to have a typo.